__tests__/app.test.js

```diff
@@ -208,3 +209,44 @@ describe('GET /api/articles', () => {
 
 
 
+describe('POST /api/articles/:article_id/comments', () => {
+  it('should respond with the posted comment', () => {
+    const article_id = 1; // Use a valid article_id
+    const newComment = {
+      username: 'butter_bridge',
+      body: 'This is a test comment',
+    };
+
+    return request(app)
+      .post(`/api/articles/${article_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        const comment = res.body.comment;
+        expect(comment).toHaveProperty('comment_id');
+        expect(comment).toHaveProperty('votes', 0);
+        expect(comment).toHaveProperty('created_at');
+        expect(comment).toHaveProperty('author', newComment.username);
+        expect(comment).toHaveProperty('body', newComment.body);
+        expect(comment).toHaveProperty('article_id', article_id);
+      });
+  });
+
+  it('should respond with status 404 for an invalid article_id', () => {
+    return request(app)
+      .post('/api/articles/9321331/comments') // Use a non-existing article_id
+      .expect(404)
+      .then((res) => {
+        expect(res.body.message).toBe('Article not found');
+      });
+  });
+
+  it('should respond with status 400 for an invalid data type in article_id', () => {
+    return request(app)
+      .post('/api/articles/sadsa/comments') // Use an invalid data type for article_id
+      .expect(400)
+      .then((res) => {
+        expect(res.body.message).toBe('Invalid article_id data type');
+      });
+  });
+});
```

Comment:
Please add another test for a bad request where properties are missing.  (e.g. no username or body properties)